homeassistant/components/services.yaml

```diff
@@ -601,3 +601,11 @@ nest:
       trip_id:
         description: Optional identity of a trip. Using the same trip_ID will update the estimation.
         example: trip_back_home
+
+zoneminder:
```

Comment:
We should move this to a services.yaml under zoneminder package that needs to be created under components. See deconz for example.

Reply:
Sounds good!